ecs-logs-collector.sh

```diff
@@ -331,17 +355,22 @@ get_docker_logs()
       ;;
     *)
       warning "The current operating system is not supported."
+      return 1
```

Comment:
I sort of wonder if, instead of trying to infer the location of the docker log based on a list of known systems, we instead react to the presense of the logs in any of these common locations.

Probably out of scope for this PR.